pyms/config/conf.py

```diff
@@ -144,3 +146,61 @@ def __verify_deprecated_env_variables(config):
         except AttrDoesNotExistException:
             pass
         logger.warning(msg)
+
+
+def create_conf_file(use_requests: bool = False, use_swagger: bool = False) -> Union[Exception, str]:  # pragma: no cover
+    """
+    Creates a configuration file defining
+
+    :param use_requests: Do you want to use requests, defaults to False
+    :type use_requests: bool, optional
+    :param use_swagger: Do you want to use swagger, defaults to False
+    :type use_swagger: bool, optional
+    :raises FileExistsError: Config file already exists
+    :raises IOError: Config file creation failed.
+    :return: Raises FileExistsError or IOError OR returns config_file_path
+    :rtype: Union[Exception, str]
+    """
+    # Try using env value for config file, if not found use default
+    CONFIG_FILE = os.getenv(CONFIGMAP_FILE_ENVIRONMENT, None)
+    if not CONFIG_FILE:
+        CONFIG_FILE = DEFAULT_CONFIGMAP_FILENAME
+    # Prevent overwriting existing file
+    if os.path.exists(CONFIG_FILE):
+        raise FileExistsError("Config file already exists at '{}'".format(os.path.abspath(CONFIG_FILE)))
+    # Create config dict
+    config = {"pyms": {}}
+    # add services
+    if use_requests:
+        if not config["pyms"].get("services", None):
+            config["pyms"]["services"] = {}
+        config["pyms"]["services"]["requests"] = {"data": ""}
+    if use_swagger:
+        if not config["pyms"].get("services", None):
+            config["pyms"]["services"] = {}
+        config["pyms"]["services"]["swagger"] = {"path": "", "file": "swagger.yaml"}
+    # add Basic Flask config
+    config["pyms"]["config"] = {
+        "DEBUG": True,
+        "TESTING": False,
+        "APP_NAME": "Python Microservice",
+        "APPLICATION_ROOT": ""
+    }
+    try:
+        with open(CONFIG_FILE, 'w', encoding='utf-8') as config_file:
+            config_file.write(yaml.dump(config, default_flow_style=False, default_style=None, sort_keys=False))
+    except Exception as ex:
+        raise ex
+    return CONFIG_FILE
+
+
+def remove_conf_file():  # pragma: no cover
```

Comment:
is remove_conf_file only used in tests? could be and extra option to force create the file or move this fuction to tests

Reply:
Moved remove_conf file to tests.common